node/test/Test/Pos/Types/Identity/BinarySpec.hs

```diff
@@ -49,3 +54,6 @@ spec = giveStaticConsts $ describe "Types" $ do
             binaryTest @T.ApplicationName
             binaryTest @T.SoftwareVersion
             binaryTest @T.BlockVersion
+        describe "Util/Data" $ do
+            binaryTest @(NewestFirst NE U)
+            binaryTest @(OldestFirst NE U)
```

Comment:
Perhaps it's better to rename this module to `Test.Pos.Core.BinarySpec` and move these tests to something like `Test.Pos.Util.ChronoSpec`.